src/careamics/config/algorithms/hdn_algorithm_model.py

```diff
@@ -0,0 +1,97 @@
+"""HDN algorithm configuration."""
+
+from typing import Literal
+
+from bioimageio.spec.generic.v0_3 import CiteEntry
+from pydantic import ConfigDict
+
+from careamics.config.algorithms.vae_algorithm_model import VAEBasedAlgorithm
+from careamics.config.architectures import LVAEModel
+from careamics.config.loss_model import LVAELossConfig
+
+HDN = "HDN"
+
+HDN_DESCRIPTION = ""
+HDN_REF = CiteEntry(
+    text='Prakash, M., Delbracio, M., Milanfar, P., Jug, F. 2022. "Interpretable '
+    'Unsupervised Diversity Denoising and Artefact Removal." The International '
+    "Conference on Learning Representations (ICLR).",
+    doi="10.1561/2200000056",
+)
+
+
+class HDNAlgorithm(VAEBasedAlgorithm):
+    """HDN algorithm configuration."""
+
+    model_config = ConfigDict(validate_assignment=True)
+
+    algorithm: Literal["hdn"] = "hdn"
+
+    loss: LVAELossConfig
+
+    model: LVAEModel  # TODO add validators
```

Comment:
Can you open an issue and state what these validators should be? Otherwise you will have to figure it out again.